src/main/java/de/dennisguse/opentracks/io/file/TrackFileFormat.java

```diff
@@ -22,7 +22,7 @@
     KML_ONLY_TRACK {
         @Override
         public TrackWriter newTrackWriter(Context context, boolean multiple) {
```

Comment:
Then `boolean multiple` is not required a parameter, right?

PS: Please format files (simples solution AndroidStudio can do this automatically before commit).

Reply:
right